### ISSUE TITLE
NAN bug

### ISSUE THREAD

**Issue by cool-RR** [brython-dev/brython#1512]
```
>>> n=float('nan')
>>> n+5
Traceback (most recent call last):
  File <string>, line 1, in <module>
ValueError: long_int argument is not a valid number: "NaN"
>>> 
```

I don't know whether it's fixed in https://github.com/brython-dev/brython/commit/df521bf891116e9e78efc48b935abd4fd496e2e5 but I see it's not in the tests, so it should be. I haven't tried other actions with NaN, probably worth trying.

**Comment by PierreQuentel** [comment]
Yes, it was fixed in df521bf, but there may be other bugs left...

**Comment by cool-RR** [comment]
Cool. I wonder whether you can take these kind of tests and more, in bulk, from cpython.
